packages/database/src/modelClasses/DataServiceSyncGroup.js

```diff
@@ -25,6 +25,35 @@ class DataServiceSyncGroupType extends DatabaseType {
       log_message: message,
     });
   }
+
+  async getLogsCount() {
+    const [{ count }] = await this.database.executeSql(
+      `
+         SELECT count(sgl.*) FROM sync_group_log sgl
+         JOIN data_service_sync_group dssg ON dssg.code = sgl.sync_group_code
+         WHERE dssg.id = ?
+       `,
+      [this.id],
+    );
+
+    return count;
+  }
```

Comment:
Tested a bit and the endpoint returns string as count. Suspect count is a string in here? 
<img width="232" alt="image" src="https://user-images.githubusercontent.com/31789355/186592616-24f979dd-f583-48d5-938c-2d6aa8cc8e29.png">

Reply:
Ah great catch! I even encountered this same bug not long ago 🤦 will fix